python/mxnet/kvstore.py

```diff
@@ -349,6 +349,88 @@ def row_sparse_pull(self, key, out=None, priority=0, row_ids=None):
             check_call(_LIB.MXKVStorePullRowSparse(
                 self.handle, mx_uint(len(ckeys)), ckeys, cvals, crow_ids, ctypes.c_int(priority)))
 
+    def set_gradient_compression(self, compression_params=None):
+        """ Specifies type of low-bit quantization for gradient compression if any,
+         and additional arguments depending on the type of compression being used.
+
+        Parameters
+        ----------
+        compression_params : dict
```

Comment:
Would this doc render correctly?

Reply:
Changed it a bit and verified
[](https://i.imgur.com/yb2XWlX.png)